.ci/renewable-deploy/image-creator.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -ex
+
+# Login to Docker Hub for pushing images into register
+echo ${DOCKER_PASSWORD} | docker login --username ${DOCKER_USERNAME} --password-stdin
+## Docker image for application php-fpm container
+docker image pull ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG_PHP_FPM} || (
+    echo "Image not found (see warning above), building it instead..." &&
+    docker image build \
+        --tag ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG_PHP_FPM} \
+        --target production \
+        --no-cache \
+        -f docker/php-fpm/Dockerfile \
+        . &&
+    docker image push ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG_PHP_FPM}
```

Comment:
I was thinking about what you said about pushing the configured image to public dockerhub.
I agree with you. And I have two possible solutions. Create a private repository on dockerhub (we can have one for free), or use Google Cloud Container registry https://console.cloud.google.com/gcr/images).

Reply:
Resolving the conversation as currently, there are no sensitive pieces of information in the docker image.